core/dbt/tracking.py

```diff
@@ -30,20 +30,22 @@
 
 COLLECTOR_URL = "fishtownanalytics.sinter-collect.com"
 COLLECTOR_PROTOCOL = "https"
+DBT_INVOCATION_ENV = "DBT_INVOCATION_ENV"
 
-INVOCATION_SPEC = "iglu:com.dbt/invocation/jsonschema/1-0-2"
-PLATFORM_SPEC = "iglu:com.dbt/platform/jsonschema/1-0-0"
-RUN_MODEL_SPEC = "iglu:com.dbt/run_model/jsonschema/1-0-2"
-INVOCATION_ENV_SPEC = "iglu:com.dbt/invocation_env/jsonschema/1-0-0"
-PACKAGE_INSTALL_SPEC = "iglu:com.dbt/package_install/jsonschema/1-0-0"
-RPC_REQUEST_SPEC = "iglu:com.dbt/rpc_request/jsonschema/1-0-1"
+ADAPTER_INFO_SPEC = "iglu:com.dbt/adapter_info/jsonschema/1-0-0"
```

Comment:
If you haven't already, I think we'll need to add these two new validation schemas to https://github.com/dbt-labs/iglu.sinter-collect.com

Doc: https://www.notion.so/dbtlabs/Add-new-metrics-to-tracking-in-dbt-3484cc31b8f449dfa2e960d2bbb15afe

Reply:
Yeppers.
https://github.com/dbt-labs/iglu.sinter-collect.com/pull/67 although there seems to be an issue with circle CI we'll need if we want to enforce the checks there.